Classes/RedirectStorage.php

```diff
@@ -138,17 +149,19 @@ protected function addRedirectByHost($sourceUriPath, $targetUriPath, $statusCode
     {
         $redirect = new Redirect($sourceUriPath, $targetUriPath, $statusCode, $host);
         $this->updateDependingRedirects($redirect);
+        $this->persistenceManager->persistAll();
```

Comment:
Like Sebastian wrote, in this case we could do
```php
$this->persistenceManager->whitelistObject($redirect);
$this->persistenceManager->persistAll(true);
```
Otherwise this can have undesired side effects